website/src/components/visualisation/Game/SampleVis.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useRef, useEffect, useState } from 'react';
+import * as d3 from 'd3';
+
+interface IProps {
+    data?: any[];
+}
+
+const SampleVis = (props: IProps) => {
+    // useRef Hook creates a variable that keeps values across rendering passes 
+    // holds the component's SVG DOM element
+    // initialised as null and React assigns it later (see return statement)
+    const d3Container = useRef(null);
+
+    useEffect(() => {
+        if (props.data && d3Container.current) {
+            const svg = d3.select(d3Container.current);
+
+            // Bind D3 data
+            const update = svg
+                .append('g')
+                .selectAll('text')
+                .data(props.data);
+
+            // Enter new D3 elements
+            update.enter()
+                .append('text')
+                .attr('x', (d, i) => i * 25)
+                .attr('y', 40)
+                .style('font-size', 24)
+                .text((d: number) => d);
```

Comment:
oh my god this is grim :(

Reply:
this is d3 in general - most other libraries including recharts are built on top of d3 because this is what it looks like so not sure how much we can do about this...

Reply:
me not know about SVGs, would be happy to be filled in later.